cluster/ubuntu/config-default.sh

```diff
@@ -37,6 +37,10 @@ export FLANNEL_NET=${FLANNEL_NET:-172.16.0.0/16}
 # Admission Controllers to invoke prior to persisting objects in cluster
 export ADMISSION_CONTROL=NamespaceLifecycle,LimitRanger,ServiceAccount,ResourceQuota,SecurityContextDeny
 
+# Path to the config file or directory of files of kubelet
+export KUBELET_CONFIG=${KUBELET_CONFIG:-""}
```

Comment:
The default value for that flag is empty, so passing it with the default value works (even though it's redundant which can be confusing). If you prefer it that way though, that's fine with me.